### ISSUE TITLE
Duck While Outputting Speech and Sounds Does Not Work

### ISSUE THREAD

**Issue by akj** [pumper42nickel/eloquence_threshold#40]
### Description
The NVDA audio ducking setting "Duck when outputting speech and Sounds does not work with Eloquence Threshold.

### Steps to Reproduce
- Assign Eloquence as speech synthesizer
- Use NVDA+Shift+D to switch audio ducking modes to "Duck when outputting speech and sounds"
- Activate a source of audio like music or a video
### Expected Behavior
NVDA ducks its volume while the other audio source is playing.

### Actual Behavior
NVDA's volume remains constant.

### NVDA and Eloquence Versions
- NVDA: 2025.2 Beta 2 (though 2025.1 does not work either)
- Eloquence Threshold Installed Version: 0.20250420.01

### Other
The "always duck" mode still works with Eloquence Threshold. Duck when outputting speech and sounds works with eSpeak and OneCore voices.
